code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -142,6 +142,7 @@
 	description = "A synthetically produced imitation of blood."
 	taste_description = "oily"
```

Comment:
i know it's not in the PR title/desc but can you change this to "oil" while you're at it cause "you taste oily" has been grating on me and i am lazy